evm/bridge-token-factory/contracts/BridgeTokenFactory.sol

```diff
@@ -39,6 +39,7 @@ contract BridgeTokenFactory is
     uint8 public omniBridgeChainId;
 
     mapping(uint128 => bool) public completedTransfers;
+    mapping(uint128 => bool) public claimedFee;
```

Comment:
As I see, we can remove `claimedFee` and just use the `completedTransfers`.
@kiseln @olga24912 do you agree?

Reply:
Actually, no, this will not work because we can't use the near nonce instead of the origin nonce, so this map is needed.

Reply:
No, we can't